src/components/MapImageControl.vue

```diff
@@ -156,11 +156,32 @@ export default {
       // wait for modal to close
       // when executed directly, the modal is visible in the export
       this.showPopUp = false
+
+      // was the width set on the container (if not we have  to set it temporary)
+      const container = this.map.getContainer()
+      // check if the width was not set
+      const widthSet = (container.style.width !== '')
       setTimeout(async () => {
+        console.log('this.map.printPlugin', this.map.printPlugin.printMap, this.map)
+
+        if (!widthSet) {
+          // Fix some minor style issues in the export
+          container.style.width = container.clientWidth + 'px'
+          container.style.height = container.clientHeight + 'px'
+          container.style.marginLeft = 0
+        }
+
         this.map.printPlugin.printMap(this.exportSize, this.name)
 
         this.map.on('easyPrint-finished', () => {
```

Comment:
is this when exporting? why do you set the style on the entire container, not on a copy for styling? Does this work properly?

Reply:
This is indeed as intended. I would rather not "clone" the node, as it has a lots of data/events connected to it and the Leaflet library is tested on cloned maps.